### PR TITLE
LPS-144161 Site search in Team Extranet

### PR DESCRIPTION
**Pull request by atjuan**
- LPS-144161 Include search bar in main navbar fragment
- LPS-144161 Configure search bar in master page
- LPS-144161 Create and configure search page

### CODE REVIEW

**Comment by atjuan** [comment]
ci:test:sf

**Comment by atjuan** [comment]
ci:test:relevant

File: modules/apps/site-initializer/site-initializer-team-extranet/src/main/zippableResources/fragments/team-extranet/fragments/main-navbar/index.css [hunk 1]
```diff
@@ -41,29 +86,38 @@
...
+textarea:-webkit-autofill:focus,
+select:-webkit-autofill,
+select:-webkit-autofill:hover,
+select:-webkit-autofill:focus {
+	-webkit-text-fill-color: var(--white);
+	-webkit-box-shadow: 0 0 0px 1000px transparent inset;
+	box-shadow: 0 0 0px 1000px transparent inset;
+	color: var(--white);
+	transition: background-color 5000s ease-in-out 0s;
 }
```
Comment (marcoscv-work) [inline modules/apps/site-initializer/site-initializer-team-extranet/src/main/zippableResources/fragments/team-extranet/fragments/main-navbar/index.css:103]: We could probably remove as much specificity by removing for example `.main-navbar` in those 5 styles

**Comment by marcoscv-work** [comment]
LGTM!

**Comment by marcoscv-work** [comment]
ci:forward

**Pull request closed without merging**